### PR TITLE
Add PHP7 to test matrix

### PR DESCRIPTION
**Pull request by lunetics**

### CODE REVIEW

**Comment by dbu** [comment]
bizzare that build error with php 7 and symfony 2.4. maybe restart and see if its something real? if it goes away, i would not put php 7 into allowed failures for now, so you see when new features would break php 7 or php 7 starts to break this bundle.

**Comment by lunetics** [comment]
Why, we have marked 2.4 as failing, so is also 7 with 2.4 failing

**Comment by dbu** [comment]
ah perfect then. 2.4 is end of live, so maybe remove it from the matrix
and add a ["conflict": { "symfony/symfony": "2.4.*"
}](https://getcomposer.org/doc/04-schema.md#conflict) to composer.json?

**Comment by lunetics** [comment]
sure, fell free to add it :)

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +26,7 @@ script: phpunit --coverage-text
 matrix:
   allow_failures:
     - env: SYMFONY_VERSION=2.4.*
+    - php: 7
```
Comment (dbu) [inline .travis.yml]: so lets remove this line as its green already.

**Comment by lunetics** [comment]
@dbu What about adding you to the travis notifications? :)

**Comment by dbu** [comment]
i'd rather not. i see the builds on pull requests, but i get so many of those notification emails i often don't look at them anymore.

**Pull request merged**